.github/workflows/pythonapp.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements.txt
-        
+
```

Comment:
This is a silly change!  I'm going to see if I can remove it before merge.

Reply:
Decided to just take the blank lines out, so now you have a different silly change.  Oh well, end result is a cleaner file so I'll take it!  (assuming github actions is working again...)